tools/release/sign_all_apks.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Get the command line parameters
+PARAM_KEYSTORE_PATH=$1
+PARAM_DIRECTORY=$2
+CHECKSUM_FILE="checksum.txt"
+
+if [[ $PARAM_KEYSTORE_PATH == ""  && $PARAM_DIRECTORY == "" ]]
+then
+  echo "Usage: ./sign_all_apks \$KEY \$FOLDER"
+  exit 1
+fi
+if [[ $PARAM_KEYSTORE_PATH == "" || $PARAM_DIRECTORY == "" ]]
+then
+  echo "Usage: ./sign_all_apks \$KEY \$FOLDER"
+  exit 1
+fi
```

Comment:
I am not sure both checks are necessary, also I suggest verifying the number of parameters using `"$#" -ne 1` in your condition